modular_zubbers/code/modules/clothing/head/wig.dm

```diff
@@ -0,0 +1,73 @@
+/obj/item/clothing/head/wig
+	var/obj/item/clothing/head/head_accessory
+	var/mutable_appearance/head_accessory_MA
+
+	var/overlay_on = FALSE
+
+	var/mob/living/carbon/holder
+	var/item_path
+
+/obj/item/clothing/head/wig/attackby(obj/item/I, mob/living/user)
```

Comment:
```suggestion
/obj/item/clothing/head/wig/attackby(obj/item/attachment, mob/living/user)
```

Reply:
Or accessory, or to_be_attached, or something like that

Reply:
Uhm, sure!